src/ai/backend/manager/models/kernel.py

```diff
@@ -109,6 +117,13 @@ class KernelStatus(CIStrEnum):
     ERROR = "ERROR"
     CANCELLED = "CANCELLED"
 
+    @classmethod
+    def with_containers(cls) -> set[KernelStatus]:
+        return {
+            cls.CREATING,
+            cls.RUNNING,
+        }
```

Comment:
"with" gives the feeling that it might be used as a context manager. Please give it a different name.